### DISCUSSION TITLE
Trojan:Win32/IcedID!pz

### DISCUSSION THREAD

**Discussion by leon-andria** [kcl-lang/kcl#1422] [General]
Trying to use the konfig module today. However, the kcl run command failed because of the following error:
`failed to compile the kcl package
LoadLibraryExW failed`

In the same time, Windows Defender Antivirus is triggered about malicious threat and report out about  potential `Trojan:Win32/IcedID!pz` during the compilation.

**Comment by Peefy** [comment]
It seems the the kcl compilation is killed by the Windows Defender, and we will try to fix it. cc @zong-zhe 

Could you please set the env KCL_FAST_EVAL=1 and try again?

**Reply by leon-andria** [reply]
I set the environment variable but no luck. Still Windows Defender is becoming crazy about it.
However, one point to note is, if I compile simple files, there is no issues from Windows Defender

**Comment by zong-zhe** [comment]
Hi @leon-andria 😄

Thanks for your feedback. When compiling kcl project, the compiler will request the local credentials to download the third-party dependencies. It may be that the behavior of requesting credentials many times causes it to be considered as a threat by the Windows Defender. I will try to do some work to fix this thing.

**Reply by leon-andria** [reply]
That would be helpful for transparency. And if I may ask, in a kind of debug mode, we could see logs. This would make users more confident of using the CLI rather than having a negative feeling of the tool.

**Reply by zong-zhe** [reply]
Thanks for the good suggestion 😄. We will support it in future versions.
